cpp/source/client/ClientManagerImpl.cpp

```diff
@@ -764,7 +764,7 @@ State ClientManagerImpl::state() const {
 }
 
 MessageConstSharedPtr ClientManagerImpl::wrapMessage(const rmq::Message& item) {
-  assert(item.topic().resource_namespace() == resource_namespace_);
+  // assert(item.topic().resource_namespace() == resource_namespace_);
```

Comment:
Why comment out this assertion?

Reply:
The resource_namespace_ is always "", this causes this assert to fail.

Reply:
@yulangz It's true `resource_namespace_` is empty for the moment. But the assertion should pass